Website/Packages/datax-query/src/modules/query/queryActions.js

```diff
@@ -0,0 +1,112 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+import Q from 'q';
+import { getApiErrorMessage } from 'datax-common';
+import * as Api from './api';
+import * as KernelActions from './kernelActions';
+import * as KernelSelectors from './kernelSelectors';
+
+/**
+ *
+ * REDUX Action Types
+ *
+ */
+
+// Init
+export const QUERY_INIT = 'QUERY_INIT';
+export const QUERY_NEW = 'QUERY_NEW';
```

Comment:
There is currently no "action" functions using QUERY_INIT and QUERY_NEW, you should remove this dead code

Reply:
Updated